Docs/Data Sharing Framework/Creating Custom Components.md

```diff
@@ -0,0 +1,55 @@
+Creating Custom Components
+==========================
+
+Create the Component
+--------------------
+
+1. Create a LabVIEW project for your component(s).
+
+**Note**: It's recommended that you create two projects: Development and Build. Because this framework uses classes, they won't be editable when loaded under multiple target contexts. Use the Build project to include all your supported target types, which will lock the classes, and the Development project with just 'My Computer' to allow the classes to be edited.
+
+2. Add a LabVIEW library to the project. This is the library that will be built into the components packed project library (PPL), and the name of this library will be the name of your component.
+
+3. Add DSF Core.lvlibp to the project.
+
+**Note**: There is a different Framework PPL for each target type supported by the Framework, since PPLs are binaries built against specific runtimes and CPU architectures. To ensure component classes don't break when building for different target types, the name of the core Framework Core PPL is always 'DSF Core.lvlibp' to ensure class inheritance does not break across all targets. When building for multiple target types, the DSF Core PPL must be swapped out on disk for each target type you build for. Because of this, it is recommended to use the Windows version of the DSF Core PPL in the Development project for your component, and then use the Build project for PPL swapping and building for each target type.
```

Comment:
since this is the first time we are using acronym PPL, recommend we spell it out:
packed project library (PPL)